apps/src/componentLibrary/toggle/Toggle.tsx

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, {ChangeEvent, memo} from 'react';
 import classnames from 'classnames';
 import Typography from '@cdo/apps/componentLibrary/typography';
 import moduleStyles from './toggle.module.scss';
 
-interface ToggleProps {
+export interface ToggleProps {
+  /** Toggle checked state */
   checked: boolean;
-  onChange: () => void;
+  /** Toggle onChange handler */
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  /** The name attribute specifies the name of an input element.
+   The name attribute is used to reference elements in a JavaScript,
+   or to reference form data after a form is submitted.
+   Note: Only form elements with a name attribute will have their values passed when submitting a form. */
   name: string;
+  /** The value attribute specifies the value of an input element. */
   value?: string;
+  /** Toggle label*/
   label?: string;
+  /** Is Toggle disabled */
   disabled?: boolean;
+  /** Toggle switch placement */
   position?: 'left' | 'right';
+  /** Size of Radio Button */
   size?: 'xs' | 's' | 'm' | 'l';
```

Comment:
As a follow-up: would be great to share this type if multiple components are using it.

Reply:
yep, that was the plan, thanks